pom.xml

```diff
@@ -933,6 +940,24 @@
                 <artifactId>cassandra-driver</artifactId>
                 <version>3.1.4-1</version>
             </dependency>
+
+            <dependency>
+                <groupId>org.junit.jupiter</groupId>
+                <artifactId>junit-jupiter-api</artifactId>
+                <version>${dep.junit-jupiter.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.junit.platform</groupId>
+                <artifactId>junit-platform-launcher</artifactId>
+                <version>${dep.junit-platform.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.junit.jupiter</groupId>
+                <artifactId>junit-jupiter-engine</artifactId>
+                <version>${dep.junit-jupiter.version}</version>
+            </dependency>
```

Comment:
The changes in the previous commit are in a separate commit so that they can be removed atomically by reverting the commit once Presto is on Airbase >= 67.